code/modules/clothing/suits/jobs.dm

```diff
@@ -96,102 +96,41 @@ obj/item/clothing/suit/apron/overalls/blue
-	icon_state = "wardenbluejacket"
-	item_state = "wardenbluejacket"
+/obj/item/clothing/suit/storage/toggle/security/navywarden
+	name = "brig officer jacket"
+	desc = "A comfy NT security jacket for everyday wear. This one has the sleeve insignia of a brig officer and is blue."
+	icon_state = "jacket_warden"
+	item_state = "jacket_warden"
+	icon_open = "jacket_warden_open"
+	icon_closed = "jacket_warden"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 
-/obj/item/clothing/suit/security/navyhos
-	name = "head of security's jacket"
-	desc = "This piece of clothing was specifically designed for asserting superior authority."
-	icon_state = "hosbluejacket"
-	item_state = "hosbluejacket"
+/obj/item/clothing/suit/storage/toggle/security/navyhos
+	name = "head of security jacket"
+	desc = "A comfy NT security jacket for everyday wear. This one has the sleeve insignia of a head of security and is blue."
+	icon_state = "jacket_hos"
+	item_state = "jacket_hos"
+	icon_open = "jacket_hos_open"
+	icon_closed = "jacket_hos"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 
-//Detective
-
-/obj/item/clothing/suit/storage/toggle/det_jacket
-	name = "detective's jacket"
-	desc = "Stylish yet comfortable professional jacket manufactured by CL corporation for NT detectives. Unique fiber structure will offer moderate protection from various hazards investigators may encounter in the line of duty"
-	icon = 'icons/obj/clothing/detective.dmi'
-	icon_state = "det"
-	item_state = "det"
-	blood_overlay_type = "coat"
-	contained_sprite = TRUE
-	body_parts_covered = UPPER_TORSO|ARMS
-	allowed = list(/obj/item/tank/emergency_oxygen, /obj/item/device/flashlight,/obj/item/gun/energy,/obj/item/gun/projectile,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/handcuffs,/obj/item/storage/fancy/cigarettes,/obj/item/flame/lighter,/obj/item/device/taperecorder)
-	armor = list(melee = 50, bullet = 10, laser = 25, energy = 10, bomb = 0, bio = 0, rad = 0)
-	siemens_coefficient = 0.7
-
-/obj/item/clothing/suit/storage/toggle/det_trench
-	name = "brown trenchcoat"
-	desc = "A rugged canvas trenchcoat, designed and created by TX Fabrication Corp. The coat is externally impact resistant - perfect for your next act of autodefenestration!"
-	icon_state = "detective"
-	item_state = "detective"
-	blood_overlay_type = "coat"
-	icon_open = "detective_open"
-	icon_closed = "detective"
-	body_parts_covered = UPPER_TORSO|ARMS
-	allowed = list(/obj/item/tank/emergency_oxygen, /obj/item/device/flashlight,/obj/item/gun/energy,/obj/item/gun/projectile,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/handcuffs,/obj/item/storage/fancy/cigarettes,/obj/item/flame/lighter,/obj/item/device/taperecorder)
-	armor = list(melee = 50, bullet = 10, laser = 25, energy = 10, bomb = 0, bio = 0, rad = 0)
-	sprite_sheets = list("Vox" = 'icons/mob/species/vox/suit.dmi')
-	siemens_coefficient = 0.7
-
-/obj/item/clothing/suit/storage/toggle/det_trench/black
```

Comment:
This is still mapped in. Will have to update map to appease Travis if you're deleting this path. Also don't forget to map in any replacements.

Reply:
How do I find out where it's mapped in? I'm not sure how to remove it, or replace it.

Reply:
I don't THINK Travis gives an exact location. It just gave me: _Invalid type in map. /obj/item/clothing/suit/storage/toggle/det_trench/black_

I'd probably check the locker in the detective's officer as my first guess. Unfortunately I'm not too familiar with the mapping tools so I'm not sure if there's a way to search item paths in there. Hopefully someone more familiar can help.

Reply:
Alright, thank you. I'll ask around.

Reply:
I figured it out, and removed it. The placement was for an easter egg with skeletons in the walls. So, no replacement.